src-ts/types.ts

```diff
@@ -0,0 +1,28 @@
+export interface DeployedContracts {
+  l1UpgradeExecutorLogic: string;
+  l2TimelockLogic: string;
+  l2GovernorLogic: string;
+  l2FixedDelegateLogic: string;
+  l2TokenLogic: string;
+  l2UpgradeExecutorLogic: string;
+  l1GovernanceFactory: string;
+  l2GovernanceFactory: string;
+  l1ReverseCustomGatewayLogic: string;
+  l1ReverseCustomGatewayProxy: string;
+  l2ReverseCustomGatewayLogic: string;
+  l2ReverseCustomGatewayProxy: string;
+  l1TokenLogic: string;
+  l1TokenProxy: string;
+  l2CoreGoverner: string;
+  l2CoreTimelock: string;
+  l2Executor: string;
+  l2ProxyAdmin: string;
+  l2Token: string;
+  l2TreasuryGoverner: string;
+  l2ArbTreasury: string;
+  arbitrumDAOConstitution: string;
```

Comment:
```suggestion
  l2ArbitrumDAOConstitution: string;
```

Reply:
i kinda like keeping this name in like with the sol contract name